apps/products-feed/src/lib/metadata-manager.ts

```diff
@@ -0,0 +1,93 @@
+import { MetadataEntry, EncryptedMetadataManager } from "@saleor/app-sdk/settings-manager";
```

Comment:
Doubled metadata mgr (the second one is in the app-configuration). One of them should be removed